src/styles/UserProfilePage.css

```diff
@@ -28,6 +23,31 @@
   font-size: 16px;
 }
 
+.user-profile-settings-header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.user-profile-settings-header h2 {
+  color: var(--pp-blue);
+  font-weight: 900;
```

Comment:
import fonts PR should handle this; there shouldnt be need to overwrite the default